README.md

```diff
@@ -12,26 +12,26 @@ This was initially built and is intended to work on [Heroku](http://heroku.com)
 
 ## Upgrading?
 
-If you are upgrading from a version of asset_sync **< 0.2.0** (i.e. 0.1.x). All of the references to config variables have changed to reference those used in **Fog**. Ensure to backup your `asset_sync.rb` or `asset_sync.yml` files and re-run the generator. You may also then need to update your ENV configuration variables (or you can change the ones that are referenced).
+Upgraded from 1.x? Read `UPGRADING.md`
 
 ## Installation
 
 Add the gem to your Gemfile
 
 ``` ruby
-gem 'asset_sync'
+gem "asset_sync"
```

Comment:
I think that can be removed? Because you always need to add a provider which is described below?